backend/routes/passwordRoute.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const router = express.Router();
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+router.post('/check-password', async(req, res) => {
+    const {password} = req.body;
+    const findPassword = await prisma.user.findUnique({
+        where: {password}
+    })
+
+    const exists = findPassword !== null
+    res.json({exists});
+})
+
+router.put('/', async(req, res) => {
```

Comment:
You don't have to create another endpoint for this logic. You can include this in the above route that you defined.